code/game/objects/structures/tables_racks.dm

```diff
@@ -219,6 +220,10 @@
 
 /obj/structure/table/attack_hand(mob/living/user)
 	user.changeNext_move(CLICK_CD_MELEE)
+	if(tableclimber)
+		tableclimber.Stun(2)
+		visible_message("You've been knocked off the table!")

```

Comment:
This will show the message you've been knocked off the table for all users in the area.

If you look at the definition of the visible message proc
visible_message(message, self_message, blind_message)

You actually want
table_climber.visible_message("[tableclimber.name] has been knocked off the table", "You've been knocked off the table")

This then will show the tableclimber user the "You've been ..." and all other users within standard hear range the "Person has been knocked off the table"